packages/perspective-viewer-d3fc/src/js/charts/heatmap.js

```diff
@@ -31,8 +31,11 @@ function heatmapChart(container, settings) {
     const yDomain = crossAxis
         .domain(settings)
         .settingName("splitValues")
-        .valueName("mainValue")(data)
-        .reverse();
+        .valueName("mainValue")(data);
+    if (settings.splitValues.find(d => d.type !== "integer" && d.type !== "float")) {
```

Comment:
I know you like to avoid IFs!, did you consider something like the following?
`settings.splitValues.some(d => d.type !== "integer" && d.type !== "float") && yDomain.reverse();`

Reply:
I was thinking about that